JAMath.py

```diff
@@ -23,9 +23,12 @@ def __init__(self, handle):
         
         self.jamath_activity = Game()
         self.build_toolbar()
-        self._pygamecanvas = sugargame.canvas.PygameCanvas(self)
+        self._pygamecanvas = sugargame.canvas.PygameCanvas(self,
+            main= self.jamath_activity.run,
```

Comment:
Code here does not conform pep8.

Code in `run` does not conform to Sugargame test activity;
* includes a pygame.init (it should not),
* does not stop pygame animation on use of GTK stop button,
* does not return on quit,

Please review the changes to the Sugargame test activity since v1.1.  These should be applied.